app/model/editions/templates/DailyEdition.scala

```diff
@@ -580,6 +598,22 @@ object FrontFoodObserver {
   )
 }
 
+object FrontSpecialSpecial3 {
+  val collectionSpecialSpecial3 = CollectionTemplate(
+    name = "Special",
+    prefill = None,
+    presentation = TemplateDefaults.defaultCollectionPresentation
+  )
+
+  val front = FrontTemplate(
+    name = "Special 3",
+    collections = List(collectionSpecialSpecial3),
+    presentation = TemplateDefaults.defaultFrontPresentation,
```

Comment:
Is this right? A special front with default formatting?

Reply:
Yup - only the two that straddle top stories get the "special" swatch. I suppose it is a bit confusing, but I'd rather not change the special names.